egs/aishell/s10/chain/device_utils.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python3
+
+# Copyright 2019-2020 Mobvoi AI Lab, Beijing, China (author: Fangjun Kuang)
+# Apache 2.0
+import numpy as np
+import torch
+
+
+def allocate_gpu_devices(num: int = 1):
+    """
+    If cuda cards are in 'Exclusive_Process' mode, we can 
+    try to get one device by putting a tensor on it. 
+    """
+    device_list = []
+    if not torch.cuda.is_available():
+        return dev_list
+    for i in range(torch.cuda.device_count()):
+        if len(device_list) >= num:
+            break
+        try:
+            device = torch.device("cuda:" + str(i))
+            tensor = torch.zeros(1)
```

Comment:
The comment at the begining of this function is too far from these two statements.